core/modules/user/user.theme.inc

```diff
@@ -386,3 +386,68 @@ function template_preprocess_user_profile(&$variables) {
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid;
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid . '__' . $view_mode;
 }
+
+/**
+ * Preprocess variables for user-simplified-page.tpl.php.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: The form structure to be rendered.
+ *   - links: A render array containing the links to other related forms.
+ *
+ * @see user-simplified-page.tpl.php
+ */
+function template_preprocess_user_simplified_page(&$variables) {
+  $form = $variables['form'];
+  if ($form['#form_id'] == 'user_login') {
+    $page_title = t('Login');
+  }
+  elseif ($form['#form_id'] == 'user_pass') {
+    $page_title = t('Reset password');
+  }
+  elseif ($form['#form_id'] == 'user_register_form') {
+    $page_title = t('Create new account');
+  }
+  
+  $variables['user_login_appearance_setting'] = config_get('system.core', 'user_login_appearance_setting');
+  $variables['user_login_page_simplified'] = $variables['user_login_appearance_setting'] == 'simplified';
+  $variables['logo_image'] = '';
+
+  if ($variables['user_login_page_simplified']) {
+    if ($logo = backdrop_get_logo()) {
+      $variables['logo_image'] = theme('image', array(
+        'path' => $logo,
+        'alt' => $page_title,
+      ));
+    }
+    $variables['site_name'] = config_get_translated('system.core', 'site_name');
+    $variables['front_page'] = url();
+    $variables['page_title'] = $page_title;
+    $variables['messages'] = theme('status_messages');
+  }
```

Comment:
Same thing, `if` statement can be dropped here:

```suggestion
  if ($logo = backdrop_get_logo()) {
    $variables['logo_image'] = theme('image', array(
      'path' => $logo,
      'alt' => $page_title,
    ));
  }
  $variables['site_name'] = config_get_translated('system.core', 'site_name');
  $variables['front_page'] = url();
  $variables['page_title'] = $page_title;
  $variables['messages'] = theme('status_messages');
```

Reply:
Removed.